vector/src/main/java/im/vector/riotx/features/home/room/detail/RoomDetailFragment.kt

```diff
@@ -1121,7 +1125,7 @@ class RoomDetailFragment @Inject constructor(
                 showSnackWithMessage(getString(R.string.copied_to_clipboard), Snackbar.LENGTH_SHORT)
             }
             is EventSharedAction.Redact                     -> {
-                promptReasonToRedactEvent(action.eventId)
+                promptConfirmationToRedactEvent(action.eventId, action.askForReason)
```

Comment:
Maybe better to pass `action` as parameter, as it is done for `promptReasonToReportContent()`